package.json

```diff
@@ -59,13 +64,13 @@
         "lint:js": "eslint .",
         "new": "node scripts/new-rule",
         "postversion": "git push && git push --tags",
+        "prepack": "tsc --emitDeclarationOnly",
```

Comment:
What I typically do is to have a dedicated type generation config instead of doing command line options in the generation: https://github.com/voxpelli/buffered-async-iterable/blob/main/declaration.tsconfig.json

I also typically run it on `prepublish` instead of `prepack`: https://github.com/voxpelli/buffered-async-iterable/blob/f77358b649574a98e4286668a6a1f5f41273a64c/package.json#L44 Why `prepack`?

Reply:
This is because I like to see the bundle using `npm pack` then inspecting the tarball

Reply:
Really good point! I will switch to that as well

For my own and other's future reference, taken from: https://docs.npmjs.com/cli/v10/using-npm/scripts#life-cycle-operation-order

![Skärmavbild 2024-04-02 kl  15 38 11](https://github.com/eslint-community/eslint-plugin-n/assets/34457/2547c146-1a39-4001-8175-04cd7d984ec5)